### DISCUSSION TITLE
Can we use more than one glide in a page

### DISCUSSION THREAD

**Discussion by Khanhan2k2** [glidejs/glide#622] [Q&A]
When i create 2 glide in my page, just one can work, Please tell me, can I use more than one glide in my page. Thanks

**Comment by MythicOrange** [comment]
Yes, you can, relatively easily, my process is that I give an ID to the containing div, so i can target it with the JS, then I do something like this in my js file;
```
if (document.getElementById('IDofTheDiv') !== null){
    var glideIDofTheDiv = new Glide('.IDofTheDiv', {
        type: 'carousel',
```

this is also so that one can be safe in not having ALL the glides displayed at all times without their javascript breaking down.
